packages/dashboard/src/components/tooltips.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
Let's use the singular for this component to keep name consistency. `tooltips` -> `tooltip`